spec/spec_helper.rb

```diff
@@ -43,7 +43,8 @@
   # inherited by the metadata hash of host groups and examples, rather than
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
-
+ 
+  config.example_status_persistence_file_path = "spec/config.example_status_persistence_file_path.txt"
```

Comment:
Is this setup correctly? @gabrielgaraujo

Reply:
Yes, it is! 🍻